### PR TITLE
Aggregator Optimizations guide update

### PR DESCRIPTION
**Pull request by jessegoodier**
## Related issue #

add DB_BUCKET_REFRESH_INTERVAL

simplify config and add link anchors

### CODE REVIEW

File: install-and-configure/install/multi-cluster/federated-etl/aggregator.md [hunk 1]
```diff
@@ -59,17 +59,20 @@ serviceAccount:
...
+
+For larger deployments of Kubecost, Aggregator can be tuned.
+The settings below are in addition to the basic configuration above.
 
 ```yaml
 kubecostAggregator:
-  replicas: 1
-  deployMethod: statefulset
-  cloudCost:
-    enabled: true
```
Comment (michaelmdresser) [inline install-and-configure/install/multi-cluster/federated-etl/aggregator.md:69]: Why remove these?

File: install-and-configure/install/multi-cluster/federated-etl/aggregator.md [hunk 2]
```diff
@@ -82,26 +85,13 @@ kubecostAggregator:
...
-  clusterName: YOUR_CLUSTER_NAME
-  cloudIntegrationSecret: cloud-integration
-  productKey:
-    enabled: true
-    key: YOUR_KEY
-prometheus:
-  server:
-    global:
-      external_labels:
-        cluster_id: YOUR_CLUSTER_NAME
```
Comment (michaelmdresser) [inline install-and-configure/install/multi-cluster/federated-etl/aggregator.md:100]: Some of these are pretty useful to be aware of; why remove them? Just to focus on Aggregator config?

File: install-and-configure/install/multi-cluster/federated-etl/aggregator.md [hunk 3]
```diff
@@ -59,17 +59,30 @@ serviceAccount:
...
+{% hint style="warning" %}
+Aggregator is a memory and disk-intensive process. Ensure that your cluster has enough resources to support the configuration below.
+
+Because the Aggregator PV is relatively small, the least expensive performance gain will be to move the storage class to a faster SSD this varies by provider, the terms used are gp3/extreme/premium/etc.
+{% endhint %}
+
+{% hint style="warning" %}
+{% endhint %}
+
+The settings below are in addition to the basic configuration above.
```
Comment (jessegoodier) [inline install-and-configure/install/multi-cluster/federated-etl/aggregator.md:75]: @michaelmdresser

Comment (jessegoodier) [reply install-and-configure/install/multi-cluster/federated-etl/aggregator.md:75]: The settings below are in addition to the basic configuration above

Comment (michaelmdresser) [reply install-and-configure/install/multi-cluster/federated-etl/aggregator.md:75]: Ah! Thank you

**Review by michaelmdresser — APPROVED** [review]

**Pull request merged**